### PR TITLE
Allow additional terminating properties to be configured

### PR DESCRIPTION
**Pull request by whamondg**
Related to #18

### CODE REVIEW

File: tests/lib/rules/terminating-properties.js [hunk 1]
```diff
@@ -50,5 +50,18 @@ ruleTester.run('terminating-properties', rule, {
     errors: [{
       message: '"to.exist" used as function'
     }]
+  },  {
+    options: [{terminators:['something', 'somethingElse']}],
```
Comment (Turbo87) [inline tests/lib/rules/terminating-properties.js]: could you change the name of that option to `properties` instead? that would match what I proposed in #20

Comment (whamondg) [reply tests/lib/rules/terminating-properties.js]: I have done that, what do you think?

Comment (Turbo87) [reply tests/lib/rules/terminating-properties.js]: sorry, I missed that

**Comment by Turbo87** [comment]
nice work, thanks @whamondg!

**Comment by Turbo87** [comment]
I just noticed that we haven't documented the option though. Could you maybe work on a follow-up PR that adds a few words about the new option to the `README`?

**Pull request merged**